### ISSUE TITLE
I wonder train on a mixture  ?  or train bridge dataset , then train on fractal?

### ISSUE THREAD

**Issue by LukeLIN-web** [allenzren/open-pi-zero#39]
`19k gradient steps with bridge (~12 epochs) and 30k with fractal (~8 epochs`

I wonder train on a mixture  ?  or train bridge dataset , then train on fractal?

**Comment by LukeLIN-web** [comment]
I think it is  [ only. train bridge then. test on Simpler windowX]  ,  or   [  only. train fractal then  test on simpler google robot]. , right ?

**Comment by LukeLIN-web** [comment]
And how to use `bridge data is 5Hz and fractal data is 3Hz.` information in simpler? did we neeed to change some hyperparameter in simpler?

**Comment by LukeLIN-web** [comment]
I trained 12hours with 8 h100, per_device_batch_size:32  it still bad on windowX robot . Don't know how to achieve readme said 12hours achieve good results ,  maybe because I don't use torch copmile.  


<img width="1694" alt="Image" src="https://github.com/user-attachments/assets/1b6c7301-f347-4411-9418-2c43022af2a8" />

**Comment by LukeLIN-web** [comment]
I have trained 48 hours in fractal with 4 h100.   But it  It only has 1% in pick coke can. 

<img width="1780" alt="Image" src="https://github.com/user-attachments/assets/a410aff9-51e2-4720-8b60-e0b45ed5828e" />
